pkgs/development/python-modules/apache-airflow/default.nix

```diff
@@ -0,0 +1,181 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, fetchFromGitHub
+, alembic
+, configparser
+, croniter
+, dill
+, flask
+, flask-appbuilder
+, flask-admin
+, flask-caching
+, flask_login
+, flask-swagger
+, flask_wtf
+, flask-bcrypt
+, funcsigs
+, future
+, GitPython
+, gunicorn
+, iso8601
+, json-merge-patch
+, jinja2
+, ldap3
+, lxml
+, markdown
+, pandas
+, pendulum
+, psutil
+, pygments
+, python-daemon
+, python-dateutil
+, requests
+, setproctitle
+, snakebite
+, sqlalchemy
+, tabulate
+, tenacity
+, text-unidecode
+, thrift
+, tzlocal
+, unicodecsv
+, werkzeug
+, zope_deprecation
+, enum34
+, typing
+, nose
+, isPy27
+}:
+
+let # import error "from pendulum import Pendulum" due to 2.x
+    pendulum1 = pendulum.overrideAttrs (super: rec {
```

Comment:
We cannot have multiple versions in `python-packages.nix`

Reply:
Having the restriction that only 1 version is allowed. Should I create a patch that has support for the newer versions?

Reply:
Yes, to get this in it needs to support the pendulum version we have in the package set.

Reply:
To get around this restriction could I treat airflow as an application rather than a library? I think this could possibly be a hard patch to maintain

Reply:
And not include it in pythonPackages